wix/windows-icu.wxs

```diff
@@ -36,12 +32,8 @@
     <SetDirectory Id="WINDOWSVOLUME" Value="[WindowsVolume]" />
 
     <!-- Components -->
-    <DirectoryRef Id="ICU_USR_BIN">
-      <Component Id="ICU_RUNTIME" Guid="af2bccab-70c7-41cd-9ec6-a265f79c0d63">
-        <File Id="ICUDT64_DLL" Source="$(var.ICU_ROOT)\Library\icu-64\usr\bin\icudt64.dll" Checksum="yes" />
-        <File Id="ICUIN64_DLL" Source="$(var.ICU_ROOT)\Library\icu-64\usr\bin\icuin64.dll" Checksum="yes" />
-        <File Id="ICUUC64_DLL" Source="$(var.ICU_ROOT)\Library\icu-64\usr\bin\icuuc64.dll" Checksum="yes" />
-      </Component>
+    <DirectoryRef Id="ICU">
+      <Merge Id="icu-msm" Language="1033" DiskId="1" SourceFile="$(var.windows-icu-msm.TargetPath)" />
```

Comment:
Why not handle this like the runtime and merge into `Library` rather than merge into `Library\icu-64`?